bitx.go

```diff
@@ -77,11 +77,12 @@ type ticker struct {
 	Bid       string `json:"bid"`
 	Ask       string `json:"ask"`
 	Last      string `json:"last_trade"`
+	Volume    string `json:"rolling_24_hour_volume"`
 }
 
 type Ticker struct {
-	Timestamp      time.Time
-	Bid, Ask, Last float64
+	Timestamp              time.Time
+	Bid, Ask, Last, Volume float64
```

Comment:
Please can you rename the field to Volume24H so that's it's clear that this is the 24 hour volume?